samples/sdl/main.c

```diff
@@ -290,7 +287,7 @@ void demo(void)
         SDL_WINDOW_SHOWN);
     if(window == NULL)
     {
-        debugPrint( "Window could not be created!\n");
+        debugPrint("Window could not be created!\n");
```

Comment:
Unintended change. Will let it stay as the extra whitespace was probably unintentional.